ast/src/expression/mod.rs

```diff
@@ -17,14 +17,14 @@ pub mod var;
 
 #[derive(Debug, Clone, PartialEq)]
 pub enum Expr {
-    Apply(apply::Apply),
+    Apply(Box<apply::Apply>),
```

Comment:
Yes, because this only requires us to box expressions that contain other expressions, instead of boxing _all_ expressions. See the changes in the test ASTs: https://github.com/encody/rusteal/pull/6/files#diff-ffc1ea321745ed83b222616a9b99836e551e877dc3952d09bd80fa2cbf84a8d4R120-R124 (-3 `Box::new` calls)